clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/model/KubernetesLoadBalancer.groovy

```diff
@@ -26,6 +27,26 @@ class KubernetesLoadBalancer implements LoadBalancer, Serializable {
   String name
   String type = "kubernetes"
   String region
+  String namespace
   String accountName
-  Set<Map<String, Object>> serverGroups
+  Service service
+  Set<Map<String, Object>> serverGroups = [] as Set
+
+  KubernetesLoadBalancer(String name, String namespace, String accountName) {
+    this.name = name
+    this.namespace = namespace
+    this.region = namespace
+    this.accountName = accountName
+  }
+
+  KubernetesLoadBalancer(Service service, List<KubernetesServerGroup> serverGroupList, String accountName) {
+    this.service = service
+    this.name = service.metadata.name
+    this.namespace = service.metadata.namespace
+    this.region = this.namespace
+    this.accountName = accountName
+    this.serverGroups = serverGroupList.collect { KubernetesServerGroup serverGroup
```

Comment:
Doesn't work in this case since we are returning a set of maps, not just a single map.